### PR TITLE
User story9

### PR DESCRIPTION
**Pull request by zachlp2016**
Added email confirmation method and verified the test was passing.
Added tests in models and new user spec.
Added test for password confirmation.

### CODE REVIEW

**Comment by WHomer** [comment]
User Story 9, Registration Email must be unique #74

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -31,6 +34,12 @@ def edit
 
   private
 
+  def email_confirmation
+    User.email_string.all? do |email|
```
Comment (BrennanAyers) [inline app/controllers/users_controller.rb:38]: Could you just use an `.include?` here?

**Review by WHomer — APPROVED** [review]
Looks like it is meeting the US.  Nice work.

File: spec/models/user_spec.rb [hunk 2]
```diff
@@ -19,4 +19,15 @@
...
   end
+
+  describe 'class methods' do
+
+    it 'email_string' do
+      user_1 = User.create!(name: "default_user", role: 0, active: true, password_digest: "8320280282", address: "333", city: "Denver", state: "CO", zip: "80000", email: "default_user@gmail.com" )
+      user_2 = User.create!(name: "default_user1", role: 0, active: true, password_digest: "8320280282", address: "333", city: "Denver", state: "CO", zip: "80000", email: "default_user1@gmail.com" )
+      user_3 = User.create!(name: "default_user2", role: 0, active: true, password_digest: "8320280282", address: "333", city: "Denver", state: "CO", zip: "80000", email: "default_user2@gmail.com" )
+
+      expect(User.email_string).not_to eq(["default_user@gmail.com", "default_user@gmail.com", "default_user@gmail.com"])
```
Comment (BrennanAyers) [inline spec/models/user_spec.rb:30]: Should we have a positive expectation as well?

**Pull request closed without merging**